docker-compose.yml

```diff
@@ -98,13 +98,69 @@ services:
       VIRTUAL_HOST: ${MATTERMOST_HOST}
       VIRTUAL_NETWORK: "nginx-proxy"
       LETSENCRYPT_HOST: ${MATTERMOST_HOST}
-      LETSENCRYPT_EMAIL: "<no value>"
+      LETSENCRYPT_EMAIL: ${LETSENCRYPT_EMAIL:-<no value>}
       EDITION: "team"
   mattermost_bridge:
     restart: unless-stopped
     image: 42wim/matterbridge:1.13.1
     volumes:
       - ./matterbridge.toml:/matterbridge.toml
+  woocommerce_shop:
+    image: btcpayserver/docker-woocommerce:3.0.6
+    environment:
+      WOOCOMMERCE_HOST: ${WOOCOMMERCE_SHOP_HOST}
+      WORDPRESS_DB_HOST: mariadb_shop
+      WORDPRESS_DB_USER: wordpress
+      WORDPRESS_DB_PASSWORD: wordpress
+      LETSENCRYPT_HOST: ${WOOCOMMERCE_SHOP_HOST}
+      LETSENCRYPT_EMAIL: ${LETSENCRYPT_EMAIL:-<no value>}
+      VIRTUAL_HOST: ${WOOCOMMERCE_SHOP_HOST}
+    expose:
+      - "80"
+    links:
+      - mariadb_shop
+    volumes:
+      - "woocommerce_shop_html:/var/www/html"
+  woocommerce_blog:
+    image: btcpayserver/docker-woocommerce:3.0.6
+    environment:
+      WOOCOMMERCE_HOST: ${WOOCOMMERCE_BLOG_HOST}
+      WORDPRESS_DB_HOST: mariadb_blog
+      WORDPRESS_DB_USER: wordpress
+      WORDPRESS_DB_PASSWORD: wordpress
+      LETSENCRYPT_HOST: ${WOOCOMMERCE_BLOG_HOST}
+      LETSENCRYPT_EMAIL: ${LETSENCRYPT_EMAIL:-<no value>}
+      VIRTUAL_HOST: ${WOOCOMMERCE_BLOG_HOST}
+    expose:
+      - "80"
+    links:
+      - mariadb_blog
+    volumes:
+      - "woocommerce_blog_html:/var/www/html"
+
+  mariadb_shop:
```

Comment:
no need of having 2 mariadb, can just use one maria server but different DB.

Reply:
Are they just created on demand?

Reply:
yes